### DISCUSSION TITLE
Dodanie metody PUT /order/checkout-forms/{id}/invoices

### DISCUSSION THREAD

**Discussion by JamesFordV3** [allegro/allegro-api#6720] [Suggestions - rejected]
Dzień dobry, 

Jak w temacie z problemem: #6717

Zgłaszam prośbę o dodanie metody:
**PUT /order/checkout-forms/{id}/invoices**

z jedną zmienną json w środu:
`{"hasExternalInvoices":true}` (or false)

w celu ustawiania statusy faktury (że została wysłana poza allegro), żeby zamówienia dla których poszła faktura poza allegro można było odhaczyć i żeby było to widoczne w przeglądarce, jak na screenie:
![obraz](https://user-images.githubusercontent.com/45076392/218732376-5f07251c-2bd2-47b5-be9d-dea49aff3d0d.png)

**Comment by JamesFordV3** [comment]
Panowie/Panie z allegro czytacie to? pytam bo 0 odpowiedzi

jest jakaś szansa że zostanie to szybko zaimplementowane? bardzo mi na tym zależy
nie ma z tym dużo roboty chyba? a widnieje mi ponad x faktur nie odhaczonych

**Reply by MartaNowaczyk** [reply]
Zbieramy tutaj informacje, a następnie przekazujemy je zespołom odpowiedzialnym za daną funkcjonalność. Na ten moment nie mam informacji czy zmiana po naszej stronie będzie możliwa. Jak tylko taka się pojawi na pewno dam znać.

**Comment by MartaNowaczyk** [comment]
Zespół odpowiedzialny za tę funkcjonalność zweryfikował opisaną kwestię i nie planuje zmian w tym zakresie. Faktury powinny iść przez allegro. Sugestię oznaczam jako odrzuconą.
